### ISSUE TITLE
Multimarkdown Engine

### ISSUE THREAD

**Issue by epitron** [locriani/magmawiki#9]
This is a pretty fancy markdown engine: http://fletcherpenney.net/multimarkdown/

It supports the basic markdown features, plus: 
- footnotes
- tables
- citations and bibliography (works best in LaTeX using BibTeX)
- math support
- automatic cross-referencing ability
- smart typography, with support for multiple languages
- image attributes
- table and image captions
- definition lists
- glossary entries (LaTeX only)
- document metadata (e.g. title, author, etc.)

Basically, the Mediawiki Markup of Markdowns. :)

It has a bunch of other awesome features too. And it's blazingly fast. How can we go wrong?

**Comment by locriani** [comment]
by eating a pizza (if that's wrong I don't want to be right)

**Comment by epitron** [comment]
You take a bold stance!

**Comment by locriani** [comment]
Problem: it's not ruby so it has a lot of dependencies we can't guarantee.
